### PR TITLE
end point 5, POST comment completed, awaiting review

### PR DESCRIPTION
**Pull request by LexJoss**

### CODE REVIEW

File: __tests__/models.test.js [hunk 1]
```diff
@@ -173,13 +173,37 @@ describe("4th endpoint, comments by review id", () => {
         })
     })
 })
-    describe.skip("5th endpoint, posting comments", () => {
+    describe("5th endpoint, posting comments", () => {
         test("5th endpoint response with a status", () => {
-            return request(app).post('/api/reviews/1/comments').expect(201)
+          return request(app).post('/api/reviews/1/comments').expect(201).send({
+            body: "blah blah blah", username : 'bainesface'})
```
Comment (two-triangles) [inline __tests__/models.test.js:179]: We could also check the structure of the posted comment to the database.

File: db/models.js [hunk 2]
```diff
@@ -49,14 +49,18 @@ const fetchCommentsByRid = (review_id) => {
...
+const postComments = (query, post) => {
+    const values = [query.review_id, post.body, post.username]
+    if (isNaN(query.review_id) || (typeof post.body) !== 'string') {return Promise.reject({status : 400, msg : "Bad Request"})}
+   
 
-    const sqlString = `SELECT * FROM comments
-    WHERE comments.review_id = $1`
+    const sqlString = `INSERT INTO comments (review_id, body, author)
+    VALUES
+    ($1, $2, $3);`
```
Comment (two-triangles) [inline db/models.js]: you can use the RETURNING constraint here that will return the inserted comment in the database, that we can then test in our testing.

**Comment by two-triangles** [comment]
Some good stuff here, I would like to see in your tests a check for what the inserted comment object should look like. Implement this in your tests and SQL query and get it passing.


Status 201, ignores unnecessary properties
Status 400, missing required field(s), e.g. no username or body properties
Status 404, username does not exist

Some extra endpoints to thing about as well. 

implement these and I will have a quick look and I think you should be good to merge :)

**Comment by two-triangles** [comment]
looks good Alex, I left a comment regarding some extra endpoints that should be tested then you will be good to go :)

**Review by aloush-dev — APPROVED** [review]

**Pull request merged**